src/components/PacksReadme/PacksReadme.tsx

```diff
@@ -218,6 +227,14 @@ export default function PacksReadme() {
       });
     }, []);
 
+    useEffect(() => {
+      if (document && fragmentIdentifier) {
+        const elementId = fragmentIdentifier.replace("#", "");
+        const parent = document.getElementById?.(elementId);
+        parent?.querySelector?.("a")?.click();
```

Comment:
Can we use `targetElement?.scrollIntoView();` if the primary purpose of the click is to scroll to the Terraform section?

Reply:
i was using scrollintoview before committing the code changes, but, it was not working.. So, i had to use click

Reply:
let me try again once